docs/components/component_overview_building_blocks_list.js

```diff
@@ -0,0 +1,99 @@
+// @flow strict
+import Box from '../graphics/building-blocks/Box.svg';
+import Column from '../graphics/building-blocks/Column.svg';
+import Container from '../graphics/building-blocks/Container.svg';
+import Flex from '../graphics/building-blocks/Flex.svg';
+import Layer from '../graphics/building-blocks/Layer.svg';
+import Letterbox from '../graphics/building-blocks/Letterbox.svg';
+import Mask from '../graphics/building-blocks/Mask.svg';
+import Pog from '../graphics/building-blocks/Pog.svg';
+import ScrollBoundaryContainer from '../graphics/building-blocks/ScrollBoundaryContainer.svg';
+import Sticky from '../graphics/building-blocks/Sticky.svg';
+import TapArea from '../graphics/building-blocks/TapArea.svg';
+import ZIndexClasses from '../graphics/building-blocks/ZIndexClasses.svg';
+import { type ListItemType } from '../pages/component_overview.js';
+
+const BUILDING_BLOCKS: ListItemType = [
+  {
+    svg: <Box />,
+    name: 'Box',
+    description:
+      'Box is a component primitive that can be used to build the foundation of pretty much any other component.',
```

Comment:
Rather than repeating this information here, can we use the existing autogen descriptions and displayNames?

Reply:
good idea  to keep  it dry. 

I tried  but I need to create a new docGen function  cos I'm  getting the following  warning.  

<img width="514" alt="Screen Shot 2022-03-24 at 7 10 37 PM" src="https://user-images.githubusercontent.com/10593890/160024676-d440d599-b46b-4ba5-8d5e-2b9c9b05e861.png">
 
I'll do it in a follow-up cleanup PR>

Reply:
Totally fine, doesn't need to be a blocker for this PR. But let's definitely explore in the near future to prevent drift and stale descriptions